src/CMakeLists.txt

```diff
@@ -373,17 +373,17 @@ endif()
 
 if( VPX_FOUND )
 	add_definitions( "-DUSE_LIBVPX=1" )
-	include_directories( "${VPX_INCLUDE_DIR}" )
+	include_directories( SYSTEM "${VPX_INCLUDE_DIR}" )
 	set( PROJECT_LIBRARIES ${PROJECT_LIBRARIES} ${VPX_LIBRARIES} )
 else()
 	message( SEND_ERROR "Could not find libvpx" )
 endif()
 
-include_directories( "${ZLIB_INCLUDE_DIR}" "${BZIP2_INCLUDE_DIR}" "${LZMA_INCLUDE_DIR}" "${JPEG_INCLUDE_DIR}" "${ZMUSIC_INCLUDE_DIR}" "${DRPC_INCLUDE_DIR}")
+include_directories( SYSTEM "${ZLIB_INCLUDE_DIR}" "${BZIP2_INCLUDE_DIR}" "${LZMA_INCLUDE_DIR}" "${JPEG_INCLUDE_DIR}" "${ZMUSIC_INCLUDE_DIR}" "${DRPC_INCLUDE_DIR}")
 
 if( ${HAVE_VM_JIT} )
 	add_definitions( -DHAVE_VM_JIT )
-	include_directories( "${ASMJIT_INCLUDE_DIR}" )
+	include_directories( SYSTEM "${ASMJIT_INCLUDE_DIR}" )
```

Comment:
We cannot use the system ASMJIT which I am assuming this implies. Our's is a modified version that is incompatible with the system libraries version.

Reply:
This doesn't change where it looks for the ASMJIT headers, it just means that your other headers will take priority in the event of a name clash, which is what you want. See `man gcc`:

```
The lookup order is as follows:

           1.  For the quote form of the include directive, the directory of the current file is searched first.

           2.  For the quote form of the include directive, the directories specified by -iquote options are searched in left-to-right order, as they appear on the command line.

           3.  Directories specified with -I options are scanned in left-to-right order.

           4.  Directories specified with -isystem options are scanned in left-to-right order.

           5.  Standard system directories are scanned.
```

Reply:
Doesn't help that the CMAKE documentation is so vague on this. If that's the case I'll simply wait for a second opinion on this merge and dismiss the change request.

Reply:
Cmake documentation is vague on purpose, because it targets multiple compilers :(

But for gcc the `SYSTEM` keyword means whether the directory will be passed as `-I` or `-isystem`, which then has the effect described above.

This PR doesn't reenable `find_package( asmjit )` so it won't accidentally use the system version of asmjit, especially because directories from `-isystem` are still checked before `/usr/include`